cfme/fixtures/v2v.py

```diff
@@ -44,23 +50,29 @@ def extract_tag(tag):
 
     if len(tags_set) < 2 or not tags_set.issubset(valid_tags):
         host_obj.remove_tags(tags=tags)
-        return True
-    return False
+        return False
+    return True
 
 
 @pytest.fixture(scope='function')
-def conversion_tags(appliance, host_creds):
+def conversion_tags(request, appliance, host_creds):
     """Assigning tags to conversation host"""
     tag1 = appliance.collections.categories.instantiate(
         display_name='V2V - Transformation Host *').collections.tags.instantiate(
         display_name='t')
+    if hasattr(request, 'param') and request.param == 'SSH':
```

Comment:
Looks like `transformation_method` could just equal the param here? Reasonable to force callers to parametrize on valid transform methods?

Reply:
I did not want to force all to send this parameter as by default VDDK is easier method(in terms of env setup) to set up and all tests can use it as default.